cmd/dump/src/lib.rs

```diff
@@ -51,7 +51,7 @@
 //! 12 hf                           0   3 recv
 //! 13 update_server                0   3 recv
 //! 14 sensor                       0   4 recv, notif: bit0(T+472)
-//! 15 host_sp_comms                0   7 recv, notif: bit0(irq82) bit1 bit2(T+59) bit3
+//! 15 host_sp_comms                0   7 recv, notif: bit0(irq82) bit1
```

Comment:
This seems like a weird thing to change; intentional?

Reply:
I objected to its columnar width. ;)